Source/WebCore/page/Navigator+LoginStatus.idl

```diff
@@ -23,12 +23,16 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+enum LoginStatus {
```

Comment:
I think you either one to rename `IsLoggedIn` in C++ to `LoginStatus`, or use `[ImplementedAs= IsLoggedIn]` here in the IDL.